.github/workflows/continuous-integration.yml

```diff
@@ -46,6 +46,12 @@ jobs:
             dependencies: "highest"
             php-version: "8.2"
 
+          # DBAL 4
+          - php-version: "8.2"
+            dependencies: "highest"
+            stability: "dev"
+            remove-orm: true
```

Comment:
you should put `remove-orm` in the definition of the matrix (with `false` as value). Otherwise, this job will be collapsed with other matrix jobs matching the other settings.

Reply:
And adding such parameter in the CI matrix requires flagging tests that require the ORM to skip them when it is not installed.

Reply:
@stof see https://github.com/doctrine/DoctrineBundle/pull/1708